### PR TITLE
fix(perf-issues): ignore graphql from consecutive db detector

### PR DESCRIPTION
**Pull request by DominikB2014**
From my EA audit, I noticed Graphql can have a bunch of different db queries that occur consecutively, the order can also change, this makes it harder to fingerprint. I've only found one EA account with graphql consecutive db issues, so its fine to just ignore graphql alltogether.

I also removed the hardcoded laracon-eu org, as it is a small change that is no longer needed.

Also, if there's a more elegant way to ignore graphql (checking different properties in the event), then lmk!

### CODE REVIEW

**Review by gggritso — COMMENTED** [review]
Could you remove the Laracon exception revert from this PR? It doesn't have anything to do with GraphQL, and besides the Laracon thing needs some more changes, like removing the `project` param from `is_event_eligible` since that was a hack

LGTM otherwise

File: src/sentry/utils/performance_issues/performance_detection.py [hunk 1]
```diff
@@ -706,7 +706,16 @@ def is_creation_allowed_for_project(self, project: Project) -> bool:
...
     @classmethod
     def is_event_eligible(cls, event, project: Project = None) -> bool:
+        request = event.get("request", None) or None
+
         sdk_name = get_sdk_name(event) or ""
+
+        if request:
+            url = request.get("url", "") or ""
+            method = request.get("method", "") or ""
+            if url.endswith("/graphql") or method.lower() in ["post", "get"]:
```
Comment (gggritso) [inline src/sentry/utils/performance_issues/performance_detection.py:716]: 🤔 why `["post", "get"]`? Are the other methods okay?

Comment (DominikB2014) [reply src/sentry/utils/performance_issues/performance_detection.py:716]: According to graphql spec, only POST and GET are used for graphql endpoints
https://graphql.org/learn/serving-over-http/#http-methods-headers-and-body

Comment (gggritso) [reply src/sentry/utils/performance_issues/performance_detection.py:716]: Ah okay roger! So, hold on, I'm little confused, wont `or method.lower() in ["post", "get"]:` ignore basically all requests, then?

Comment (DominikB2014) [reply src/sentry/utils/performance_issues/performance_detection.py:716]: Ughhh, your right, it should be `and`! I'll update and add a test for that

Comment (DominikB2014) [reply src/sentry/utils/performance_issues/performance_detection.py:716]: Very nice catch @gggritso !

Comment (gggritso) [reply src/sentry/utils/performance_issues/performance_detection.py:716]: :george_salute:

Comment (DominikB2014) [reply src/sentry/utils/performance_issues/performance_detection.py]: the or has been converted to an and.
and it ensured that non graphql post requests pass as well.

**Review by gggritso — APPROVED** [review]
Nice!

**Pull request merged**